### DISCUSSION TITLE
Logo usage permission

### DISCUSSION THREAD

**Discussion by vishalknishad** [dogecoin/dogecoin#2901] [Help needed]
I have developed an app for tracking dogecoin price. It's a free app and contribution to dogecoin community from my side.
I request you to kindly give me permission to use the official dogecoin logo.

**Comment by patricklodder** [comment]
Hi! The the logo's intellectual property is not owned by this repository; the software has only been granted usage rights. I'll try to reach out and find out who you should direct your inquiry to instead, but note that this may take some time.
